canvas_modules/harness/src/client/components/sidepanel/properties/sidepanel-properties-param-def.json

```diff
@@ -340,6 +346,16 @@
 				},
 				"control": "toggle"
 			},
+			{
+				"parameter_ref": "showHeadingDesc",
+				"label": {
+					"default": "headingDesc"
```

Comment:
Change the label to `showHeadingDesc`. Label should match with the config option.